### ISSUE TITLE
FE Skipped Headings on Register for Care Top Task Template

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10060]
## Describe the defect
The current template for the content type Register for Care is structured in such a way that the content does not have properly nested headings. 

## To Reproduce
1. Go to [Loma Linda VA Register for Care page](https://www.va.gov/loma-linda-health-care/register-for-care/)
2. View all headings on the page (you can do this using the inspector, WAVE extension, ANDI, etc.)
3. Note that headings jump from H2 > H4 skipping the H3 level heading

## AC / Expected behavior

- [ ] Facility names should be H3

## Screenshots
Skipped Heading:
![image](https://user-images.githubusercontent.com/106678594/182912078-be069499-60be-4b8d-bd73-8e23a39a6067.png)

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`
